integration-healthcheck/src/main/java/org/symphonyoss/integration/healthcheck/services/ServiceHealthIndicator.java

```diff
@@ -303,6 +318,17 @@ private Client getHttpClient() {
    */
   protected abstract ServiceName getServiceName();
 
+  /**
+   * Returns the service name to be displayed to a user.
+   * @return Friendly service name
+   */
+  protected String getFriendlyServiceName() {
+    String serviceName = getServiceName().toString();
```

Comment:
I don't think you need to make all these statements. You should just return the service name as the default behavior.

This approach becomes the health check response consistent with the older versions. HC JSON ouput is consumed by monitoring tools, so you must avoid errors in the JSON processing.

Reply:
Done